modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectFolderResourceImpl.java

```diff
@@ -148,7 +154,32 @@ public ObjectFolder putObjectFolder(
 		return _toObjectFolder(
 			_objectFolderService.updateObjectFolder(
 				objectFolder.getExternalReferenceCode(), objectFolderId,
-				LocalizedMapUtil.getLocalizedMap(objectFolder.getLabel())));
+				LocalizedMapUtil.getLocalizedMap(objectFolder.getLabel()),
+				transformToList(
+					objectFolder.getObjectFolderItems(),
+					objectFolderItem -> {
+						ObjectDefinition objectDefinition =
+							_objectDefinitionLocalService.
+								getObjectDefinitionByExternalReferenceCode(
+									objectFolderItem.
+										getObjectDefinitionExternalReferenceCode(),
+									contextUser.getCompanyId());
+
+						ObjectFolderItem serviceBuilderObjectFolderItem =
+							_objectFolderItemLocalService.
+								createObjectFolderItem(
+									new ObjectFolderItemPK(
+										objectDefinition.
+											getObjectDefinitionId(),
+										0L));
```

Comment:
Why object folder is 0L?

Reply:
Because in this case we don´t use the objectFolder, the objectFolderId already pass as parameter in `updateObjectFolder` method. But, I will put `objectFolderId` instead of `0L`.